### DISCUSSION TITLE
Make clown costumes more clowny

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6860] [Unstable feedback]
The masks are fine, but both the normal and legendary clown costumes just don't look like clown things anymore.
![image](https://user-images.githubusercontent.com/55637865/136582406-2a4a5e29-1d18-42a6-971e-b09acd033bf4.png)
![image](https://user-images.githubusercontent.com/55637865/136582433-ae5ce7b1-ca2e-4b47-a0d8-3ae6cf8d61e2.png)

They don't have the squeaky shoes yet they make the squak sound while walking, the legendary clown costume looks like just a fancy red suit, and the pants of the normal costume look like more military than anything.

another odd details are the normal costume's shoulder, the torso is white with red lines but the shoulders are red with 2 small white lines.

The suggestion is to make them pop out more as clown items and not just fancy/weirdly colored suits

**Comment by ubertpendragon** [comment]
i agree

**Comment by trevormoney1** [comment]
I feel like the clowns are fine. The legendary one has a very theatrical look to it that is pleasing. Maybe giving them proper clown shoes would help, but I don't see a major issue.

**Comment by SpazzMaticus107** [comment]
To me the costumes look fine except for the shoes.
